lib/elasticity/version.rb

```diff
@@ -1,3 +1,3 @@
 module Elasticity
-  VERSION = '2.8.0'
+  VERSION = '2.9.0'
```

Comment:
So we don't diverge from the upstream too much can we call this 2.8.0.1 or something?